reconcile/utils/glitchtip/models.py

```diff
@@ -117,7 +117,7 @@ def __hash__(self) -> int:
 class ProjectAlert(BaseModel):
     pk: int | None
     name: str
-    timespan_minutes: int
+    timespan_minutes: int = Field(alias="timespanMinutes")
```

Comment:
It must be this:
```suggestion
    timespan_minutes: int = Field(..., alias="timespanMinutes")
```
because it's a required field.